docs/source/topics/ref_minimum-system-requirements.adoc

```diff
@@ -26,9 +26,9 @@ To assign more resources to the {prod} virtual machine, see link:{crc-gsg-url}#c
 
 === {msw}
 
-* On {msw}, {prod} requires the Windows 10 Pro Fall Creators Update (version 1709) or newer.
-{prod} does not work on earlier versions or editions of {msw}.
-{msw} 10 Home Edition is not supported.
+* On {msw}, Except {msw} 10 Home Edition {prod} works on all other Windows 10 Editions.
+For Pro Edition {prod} requires the Windows 10 Pro Fall Creators Update (version 1709) or newer.
+{prod} does not work on earlier versions of Windows 10 Pro Edition.
```

Comment:
Maybe this? 
```
* On {msw}, {prod} requires the Windows 10 Fall Creators Update (version 1709) or newer.
{prod} does not work on earlier versions of {msw}.
{msw} 10 Home Edition is not supported.
```

@robin-owen ^^

Reply:
@cfergeau Your version reads more clearer, i can change it to that, but lets see what @robin-owen thinks.

Reply:
@anjannath The change @cfergeau recommends looks good to me, that's what I would change it to. :)